services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/domain/Hankealue.kt

```diff
@@ -24,4 +21,5 @@ data class Hankealue(
     var meluHaitta: Haitta13? = null,
     var polyHaitta: Haitta13? = null,
     var tarinaHaitta: Haitta13? = null,
+    var nimi: String? = null
```

Comment:
You're a bit inconsistent with the trailing commas. I think it's better to add them whenever editing the last row of a list to make future change logs cleaner.

Reply:
Good point! I added trailing commas.